Lib/test/test_coroutines.py

```diff
@@ -2191,12 +2191,21 @@ async def run_gen():
             return 'end'
         self.assertEqual(run_async(run_gen()), ([], 'end'))
 
-    def test_bpo_45813(self):
+    def test_bpo_45813_1(self):
         'This would crash the interpreter in 3.11a2'
         async def f():
             pass
         frame = f().cr_frame
-        frame.clear()
+        with self.assertWarns(RuntimeWarning):
```

Comment:
Suppress printing `RuntimeWarning: coroutine 'CoroutineTest.test_bpo_....<locals>.f' was never awaited` on the console.